spec/controllers/order_items_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe OrderItemsController, type: :controller do
+  describe "GET index" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+    end
+
+    it "renders the :index view for order items in cart" do
+       get :index
+       expect(response).to render_template("index")
+    end
+  end
+
+  describe "GET new" do
+
+    before :each do
+      @product = Product.new(id: 1, name: "some name", price: 1, user_id: 1, status: "active")
+      @item = OrderItem.new(quantity: 2, product_id: 1)
+      @product.save
+    end
+
+    it "saves a new blank instance of item in a variable" do
+      get :new, :product_id => @product.id
+      expect(@item.quantity).to eq 2
```

Comment:
I'm curious how checking the quantity of the item is at all related to a get new request